src/clj/rems/auth/oidc.clj

```diff
@@ -51,7 +51,9 @@
                                               (getx env :oidc-client-secret)]
                                  :form-params {:grant_type "authorization_code"
                                                :code (get-in request [:params :code])
-                                               :redirect_uri (str (getx env :public-url) "oidc-callback")}})
+                                               :redirect_uri (str (getx env :public-url) "oidc-callback")}
+                                 :save-request? true
+                                 :debug-body true})
```

Comment:
We could perhaps make these two configurable? Might be useful for debugging later but not as standard?

Reply:
I think these are ok to be left as standard for this call. They only affect output when an error happens.

Reply:
Ok sounds good, but where are these actually logged?